openstack/orchestration/v1/stackresources/results.go

```diff
@@ -192,21 +206,40 @@ func (r ResourceTypePage) IsEmpty() (bool, error) {
 	return len(rts) == 0, nil
 }
 
+// resourceTypes represents the type that holds the result of ExtractResourceTypes.
+// We define methods on this type to sort it before output
+type resourceTypes []string
```

Comment:
as the comment says, I added this interface simply to enable sorting. This is because the output from `heatclient` is sorted...and also because it looks _much_ nicer. However, we can possibly do the sorting in the code for the commands itself. I'm ok with either way.